configs/boards/rk3568/config-rock-3a-debian-bullseye-server-arm64-gpt.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+CMD=`realpath $0`
+CONFIG_BOARD_DIR=`dirname $CMD`
+TOP_DIR=$(realpath $CONFIG_BOARD_DIR/../../../)
+echo "TOP DIR = $TOP_DIR"
+BUILD_DIR=$TOP_DIR/build
+[[ ! -d "$BUILD_DIR" ]] && mkdir -p $BUILD_DIR
+
+# env
+export CPU=rk3568
+export BOARD=rock-3a
+export MODEL=debian
+export DISTRO=bullseye
+export VARIANT=server
+export ARCH=arm64
+export FORMAT=gpt
+export IMAGESIZE=4000MB
+
+# Add pre-installed packages for target system
+cat > $BUILD_DIR/${BOARD}-${MODEL}-${DISTRO}-${VARIANT}-${ARCH}-${FORMAT}-packages.list <<EOF
+
+EOF
+
+# Add yaml variable
+cat > $BUILD_DIR/${BOARD}-${MODEL}-${DISTRO}-${VARIANT}-${ARCH}-${FORMAT}-variable.yaml <<EOF
+{{- \$cpu := or .cpu "${CPU}" -}}
+{{- \$board := or .board "${BOARD}" -}}
+{{- \$architecture := or .architecture "${ARCH}" -}}
+{{- \$model :=  or .model "${MODEL}" -}}
+{{- \$suite := or  .suite "${DISTRO}" -}}
+{{- \$imagesize := or .imagesize "${IMAGESIZE}" -}}
+{{- \$bootpartitionend := or .bootpartitionend "1081343S" -}}
+{{- \$rootpartitionstart := or .rootpartitionstart "1081344S" -}}
+{{- \$apt_repo := or .apt_repo "radxa" -}}
+
+EOF
+
+# Add images yaml
+cat > $BUILD_DIR/${BOARD}-${MODEL}-${DISTRO}-${VARIANT}-${ARCH}-${FORMAT}-yaml.list <<EOF
```

Comment:
nit: protect against vars having spaces.

```suggestion
cat > "$BUILD_DIR/${BOARD}-${MODEL}-${DISTRO}-${VARIANT}-${ARCH}-${FORMAT}-yaml.list" <<EOF
```